zazu/util.py

```diff
@@ -137,3 +148,10 @@ def pprint_list(data):
 def raise_uninstalled(pkg_name):
     """Raises a exception for a missing package"""
     raise click.ClickException('{0} not found, install it via "apt-get install {0}" or "brew install {0}"'.format(pkg_name))
+
+
+def get_plugin_list(plugin_subclass):
+    """helper function to pull lists of plugins"""
```

Comment:
This is a useful function, though to make it more useful I think that you should return known_types instead of known_types.keys()